Description: Fix FTBFS in riscv64
 The TestBinDirVerifyImage/large_output_is_truncated tests are
 failing on riscv64 during ubuntu Launchpad builds due to a timeout error:
 time="2024-10-16T14:34:31Z" level=error msg="error flushing stdout" error="read |0: i/o timeout"
 The fix here is to increase the timeout for that test.
Author: Athos Ribeiro <athos.ribeiro@canonical.com>
Forwarded: no
Last-Update: 2024-10-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/pkg/imageverifier/bindir/bindir_test.go
+++ b/pkg/imageverifier/bindir/bindir_test.go
@@ -171,7 +171,7 @@
 		v := NewImageVerifier(&Config{
 			BinDir:             binDir,
 			MaxVerifiers:       -1,
-			PerVerifierTimeout: tomlext.FromStdTime(30 * time.Second),
+			PerVerifierTimeout: tomlext.FromStdTime(120 * time.Second),
 		})
 
 		j, err := v.VerifyImage(ctx, "registry.example.com/image:abc", ocispec.Descriptor{})
